astropy/modeling/tests/test_fitters.py

```diff
@@ -501,6 +501,21 @@ def test_param_cov(self):
         assert_allclose(fmod.parameters, beta.A.ravel())
         assert_allclose(olscov, fitter.fit_info['param_cov'])
 
+    def test_minimize(self):
+        """
+        Runs MinimizeFitter with most methods.
+        """
+        # get rid of tiny bound so the method which don't support
+        # bounds can be tested
+        self.gauss.bounds['stddev'] = (None, None)
+        for meth in ['Nelder-Mead', 'Powell', 'CG', 'BFGS', 'L-BFGS-B', 'TNC', 'SLSQP']:
```

Comment:
It may be a bit better in terms of figuring out future regressions if this was written as a pytest parametrization.

```
methods = [...]
@pytest.mark.parametrize('method',  methods)
def test_minimize(self, method):
```